src/clib/lib/res_util/subst_list.cpp

```diff
@@ -537,10 +537,10 @@ static char *trim_string(char *str) {
     return str;
 }
 
-void subst_list_add_from_string(subst_list_type *subst_list,
```

Comment:
Doc should indicate the error string is returned.

Reply:
Decided on pybind and exceptions, so this will be reverted to void